thrift/zipkincore.thrift

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+namespace cpp twitter.zipkin.thrift
```

Comment:
is this one necessary? I don't suppose you're planning to use zipkin model in C++ client.

Reply:
Honestly no. I can revert this. Just did this upfront and never went back to check what was used.